README.md

```diff
@@ -150,23 +150,29 @@ _Context: package_
 
 Use this command to run tests on a package. Currently, there are two types of tests available.
 
+#### Asset Loading Tests
```

Comment:
Thank you for adding this section.

Reply:
Never mind, I see this section was already there but is just re-sorted to the top. 😄